pages/services/edge-lb/1.2/release-notes/index.md

```diff
@@ -35,7 +35,7 @@ Released on November 27, 2018.
 
 ## Known Issues
 
-* The steps provided in the DC/OS web interface to uninstall Edge-LB are incorrect. Follow these steps in the [Edge-LB uninstall documentation](/services/edge-lb/1.2/uninstalling/).
+* The steps provided in the DC/OS 1.10 web interface to uninstall Edge-LB are incorrect. In order to correctly uninstall Edge-LB for any given DC/OS version, please follow the steps in the [Edge-LB uninstall documentation](/services/edge-lb/1.2/uninstalling/).
```

Comment:
See my comment about linking to particular edge-lb versions.

Reply:
As noted, each release will have its own release notes, so this link is correct.